### PR TITLE
[EL8 CLEANUP] Add docker runtime image and config file

### PR DESCRIPTION
**Pull request by aandvalenzuela**
Adding [apptainer](https://github.com/aandvalenzuela/cms-docker/blob/el8-cleanup/el8-cleanup/Dockerfile.runtime#L13) to runtime image due to https://github.com/aandvalenzuela/cmssw-driver-files/blob/main/images/Dockerfile.runtime#L32

### CODE REVIEW

**Comment by aandvalenzuela** [comment]
I was not sure if modifying `config.yaml` will impact the current jobs for el8

**Comment by aandvalenzuela** [comment]
> it will but only for new container e.g. just add the following group (which will hopefully built `cmssw/el8:x86_64-runtime` container)
> 
> ```
>   runtime:
>     docker: Dockerfile.${group0}
>     from: library/almalinux:8
>     variables:
>       SKIP_TESTS: true
>     groups:
>       x86_64:
>         tags:
>           ${group1}-${group0}:
> ```
> 
> once its works then we should remove the `SKIP_TESTS` and add a test for running cmssw/short runtheMatrix for this container

Right, I forgot about this part of our discussion. My bad!

File: el8/Dockerfile.runtime [hunk 1]
```diff
@@ -0,0 +1,35 @@
+FROM @BASE_IMAGE_NAME@
+LABEL maintainer="CMS Build"
+LABEL name="CMS Worker Node on EL - Runtime"
+
+ADD krb5.conf               /etc/krb5.conf
+ADD CERN.repo               /tmp/CERN.repo
+ADD RPM-GPG-KEY-kojiv2      /tmp/RPM-GPG-KEY-kojiv2
+ADD fix_ssh_config.sh       /tmp/fix_ssh_config.sh
+
+RUN dnf install -y @DEFAULT_PACKAGES@ \
```
[the remark this answers is not in the document]
Comment (aandvalenzuela) [reply el8/Dockerfile.runtime]: I have fixed EPEL_PACKAGES just in case, although this one is not empty in the config file

**Comment by aandvalenzuela** [comment]
please test

**Comment by aandvalenzuela** [comment]
please test

**Comment by smuzaffar** [comment]
I do not see bot running the tests. let merge and force built the runtime container

**Pull request merged**